packages/browser-data-collector/src/report.ts

```diff
@@ -12,12 +12,19 @@ import {
 
 export class ReportImpl implements Report {
 	id: string;
+	url: string;
 	beginning!: number;
 	end?: number;
 	data: ReportData = new Map();
 	startCollectors!: Collector[];
 	stopCollectors!: Collector[];
 
+	// Stubs that are necessary to implement but are not used.
+	body = null;
```

Comment:
Typescript's definitions now require us to implement these, but we don't seem to use them anywhere.